allergies/allergies.py

```diff
@@ -13,30 +13,18 @@
 class Allergies(object):
 
     def __init__(self, score):
-        self.score = score
-        self.allergen_list = self.lst
+        self.allergen_list = []
+        self.score = score % 256
 
-#    @property
+    @property
     def lst(self):
-        allergen_list = []
-        score = self.score % 256
         factors = 7
         while factors >= 0:
-            if score >= 2**factors:
-                score -= 2**factors
-                allergen_list.insert(0, ALLERGENS.get(2**factors))
+            if self.score >= 2**factors:
```

Comment:
I'd assign the value of `2**factors` to a variable since it's used 3 times in this method. That is not _that_ important here but it's a good habit to avoid repeating the same operation. It would be more noticeable if that would be the same DB query made 3 times.

Reply:
ok, i have implemented allergen_code so i make calculation only once